base/linalg/dense.jl

```diff
@@ -509,7 +509,7 @@ function logm(A::StridedMatrix)
         end
     end
 
-    if isreal(A) && ~np_real_eigs
+    if all(isreal, A) && ~np_real_eigs
```

Comment:
they happen to be equivalent for Bool, but I suspect this happens more out of matlab-habit than intentionally